forge-gui/res/cardsfolder/upcoming/foray_of_orcs.txt

```diff
@@ -0,0 +1,10 @@
+Name:Foray of Orcs
+ManaCost:3 R
+Types:Sorcery
+A:SP$ Amass | Type$ Orc | Num$ 2 | RememberAmass$ True | SubAbility$ DBImmediateTrig | SpellDescription$ Amass Orcs 2. When you do, CARDNAME deals X damage to target creature an opponent controls, where X is the amassed Army's power. (To amass Orcs 2, put two +1/+1 counters on an Army you control. It's also an Orc. If you don't control an Army, create a 0/0 black Orc Army creature token first.)
+SVar:DBImmediateTrig:DB$ ImmediateTrigger | ConditionCheckSVar$ X | Execute$ TrigDealDamage | TriggerDescription$ When you do, CARDNAME deals that much damage to target creature or planeswalker an opponent controls.
+SVar:TrigDealDamage:DB$ DealDamage | ValidTgts$ Creature.OppCtrl | TgtPrompt$ Select target creature an opponent controls | NumDmg$ X | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
```

Comment:
cleanup shouldn't happen delayed like this (I think I tell you this every time :P)

Reply:
you still need to Remember on the trigger

Reply:
X needs to use TriggerRemembered syntax